Makefile

```diff
@@ -112,3 +115,26 @@ verify: check format test
 
 .PHONY: verify-extended
 verify-extended: install-requirements check-generate check format test-cov test-cov-clean
+
+#####################################################################
+# Rules for cnudie component descriptors dev setup #
+#####################################################################
+
+.PHONY: cnudie-docker-images
+cnudie-docker-images:
+	@echo "Building docker images for version $(EFFECTIVE_VERSION) for registry $(CNUDIE_IMAGE_REGISTRY)"
+	@docker build -t $(CNUDIE_IMAGE_REGISTRY)/$(NAME):$(EFFECTIVE_VERSION) -f Dockerfile .
```

Comment:
Why do we have a separate docker build here instead of reusing the existing one and tagging the image as needed?
The command line option differ, e.g. with regards to memory limits. I do not like to duplicate code.

Reply:
I have changed this such that we have only one docker build target which is called with different settings.